python/lsst/obs/pfs/createDetGeom.py

```diff
@@ -0,0 +1,144 @@
+# example: createDetGeom.py
+import os
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+import lsst.utils
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        Task.__init__(self, *args, **kwargs)
```

Comment:
Since you're just calling the superclass method, you don't need to bother overriding it.